src/ray/core_worker/test/actor_manager_test.cc

```diff
@@ -20,17 +20,17 @@
 #include "ray/common/test_util.h"
 #include "ray/core_worker/reference_count.h"
 #include "ray/core_worker/transport/direct_actor_transport.h"
-#include "ray/gcs/redis_accessor.h"
-#include "ray/gcs/redis_gcs_client.h"
+#include "ray/gcs/gcs_client/service_based_accessor.h"
+#include "ray/gcs/gcs_client/service_based_gcs_client.h"
 
 namespace ray {
 
 using ::testing::_;
 
-class MockActorInfoAccessor : public gcs::RedisActorInfoAccessor {
+class MockActorInfoAccessor : public gcs::ServiceBasedActorInfoAccessor {
```

Comment:
I don't get it. If it's a mock accessor, why it has to inherit from an actual implementation? Why not inherit from `ActorInfoAccessor`? Same question for other mock classes in this PR.

Reply:
oh, yes, thanks  for the reminding